src/tests/ftest/scripts/main.sh

```diff
@@ -237,6 +237,7 @@ fi
 
 # now run it!
 # shellcheck disable=SC2086
+env
 if ! ./launch.py "${launch_args}" -th "${LOGS_THRESHOLD}" \
```

Comment:
Or alternatively set in `launch.py`'s environment when you call it:
```suggestion
if ! WITH_VALGRIND="$WITH_VALGRIND" ./launch.py "${launch_args}" -th "${LOGS_THRESHOLD}" \
```